src/sugar/autofill.ts

```diff
@@ -193,4 +200,20 @@ async function setLatestValidatedLedgerSequence(
   tx.LastLedgerSequence = ledgerSequence + LEDGER_OFFSET
 }
 
+async function checkAccountDeleteBlockers(client: Client, tx: AccountDelete) {
+  const request: AccountObjectsRequest = {
+    command: 'account_objects',
+    account: tx.Account,
+    ledger_index: 'validated',
+    deletion_blockers_only: true,
+  }
+  const response = await client.request(request)
+  if (response.result.account_objects.length > 0) {
+    throw new XrplError(
```

Comment:
I dont think throwing here will give us the right error handling flow. 

I think this will just throw and kill the event loop.

What I think we'll wanna do is to make this a promise and return a `Promise.reject()` if this fails